runtime/doc/remote_plugin.txt

```diff
@@ -101,35 +101,34 @@ the remote plugin manifest must be generated with `:UpdateRemotePlugins`.
 4. Remote plugin manifest			    *remote-plugin-manifest*
 
 Just installing remote plugins to "rplugin/{host}" isn't enough for them to be
-automatically loaded when required. The `:UpdateRemotePlugins` command must be
-executed every time a remote plugin is installed, updated, or deleted.
+automatically loaded when required. You must execute `:UpdateRemotePlugins`
+every time a remote plugin is installed, updated, or deleted.
 
-`:UpdateRemotePlugins` will generate the remote plugin manifest, a special
-vimscript file containing declarations for all vimscript entities
+`:UpdateRemotePlugins` generates the remote plugin manifest, a special
+Vimscript file containing declarations for all Vimscript entities
 (commands/autocommands/functions) defined by all remote plugins, with each
-entity associated with the host and plugin path. The manifest can be seen as a
-generated extension to the user's vimrc (it even has the vimrc filename
-prepended).
+entity associated with the host and plugin path. The manifest is a generated
+extension to the user's vimrc (it even has the vimrc filename prepended).
 
-The manifest declarations are nothing but calls to the remote#host#RegisterPlugin
-function, which will take care of bootstrapping the host as soon as the
+Manifest declarations are nothing but calls to the remote#host#RegisterPlugin
+function, which takes care of bootstrapping the host as soon as the
 declared command, autocommand, or function is used for the first time.
 
 The manifest generation step is necessary to keep Nvim's startup fast in
 situations where a user has remote plugins with different hosts. For example,
-say a user has three plugins, for Python, java and .NET hosts respectively. If
+say a user has three plugins, for Python, Java and .NET hosts respectively. If
 we were to load all three plugins at startup, then three language runtimes
-would also be spawned which could take seconds!
+would also be spawned, which could take seconds!
 
 With the manifest, each host will only be loaded when required. Continuing
-with the example, say the java plugin is a semantic completion engine for java
-source files. If it defines the autocommand "BufEnter *.java", then the java
-host will only be spawned when files ending with ".java" are loaded.
-
-If the explicit call to `:UpdateRemotePlugins` seems incovenient, try to see
-it like this: It's a way to give IDE-like capabilities to nvim while still
-keeping it fast and lightweight for general use. It can also be seen as
-analogous to the |:helptags| facility.
+with the example, say the Java plugin is a semantic completion engine for Java
+source files. If it defines the autocommand "BufEnter *.java", then the Java
```

Comment:
`source files` -> `source code` -- I'd imagine completion plugins work fine if the file hasn't been saved to disk yet.